spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/ReadConf.scala

```diff
@@ -21,14 +22,13 @@ object ReadConf {
   val DefaultFetchSize = 1000
   val DefaultConsistencyLevel = ConsistencyLevel.LOCAL_ONE
```

Comment:
Do a 
          import CassandraConnectorConf.process 

so you can simplify each 
       CassandraConnectorConf.processProperty to processProperty